extensions/visualizations/visualizations.html

```diff
@@ -21,3 +21,33 @@
     }
   </style>
 </script>
+
+<script type="text/ng-template" id="visualizations/EnumeratedFrequencyTable">
+  <strong><[options.title]></strong>
+  <table class="table">
+    <tr>
+      <th><[options.column_headers[0]]></th>
+      <th><[options.column_headers[1]]></th>
+    </tr>
+    <tr ng-repeat="row in data track by $index">
+      <td>
+        <a ng-click="toggleAnswerVisibility($index)">#<[$index + 1]></a>
+        <table class="table" ng-show="answerVisible[$index]">
+          <tr ng-repeat="item in row.answer">
+            <td ng-bind-html="item"></td>
```

Comment:
FYI I think we usually use angular-html-bind for this (see core/templates/dev/head/directives.js)

Reply:
Done.